messages_appendix.cddl

```diff
@@ -41,18 +41,17 @@ response = (
 request-id = uint
 
 ; type key 1001
-authentication-request = {
+auth-request-hkdf-scrypt-psk = {
```

Comment:
We need to update the index.bs file at the same time to reference the new names and changes.

Reply:
> We need to update the index.bs file at the same time to reference the new names and changes.